draft-eggert-bcp45bis.md

```diff
@@ -164,12 +164,29 @@ Inappropriate postings include:
 
 # Security Considerations
```

Comment:
I think that this section should be entitled "Moderation".  This document does not need security considerations and a separate security considerations section might say just that.

Reply:
Good point. This is a leftover from BCP45, where this content was in the Security Considerations, but that need not stay that way.